MIGRATION.md

```diff
@@ -29,3 +30,113 @@ import '../../node_modules/@metamask/design-tokens/dist/styles.css';
 ```
 
 This new path points to the `dist` directory, ensuring that you're importing the most optimized and production-ready version of the stylesheet.
+
+## From version 3.0.0 to 4.0.0
+
+### Brand Colors
+
+- Grey updates
+- grey000 added
+- grey025 added
+- grey050 added
+- grey030 removed
+- grey040 removed
+- grey750 removed
+- grey1000 added
```

Comment:
In the final migration documentation, it would be beneficial to include specific code changes.  Added 
JS
Added
```
brandColor.grey000
brandColor.grey025
```
Removed
```
brandColor.grey030
```

CSS
Added
```
--brand-colors-grey-grey000 
--brand-colors-grey-grey025
```
Removed
```
--brand-colors-grey-grey030
```


This detail will aid engineers who are searching for removed tokens in the codebase.

Reply:
Yep agree - Migration doc needs some work